administrator/language/en-GB/joomla.ini

```
@@ -511,6 +518,7 @@ JGLOBAL_MODIFIED="Modified"
 JGLOBAL_MODIFIED_DATE="Modified Date"
 JGLOBAL_MOST_HITS="Most Hits"
 JGLOBAL_MOST_RECENT_FIRST="Most Recent First"
+JGLOBAL_MULTI_COLUMN_ORDER_LABEL="Multi Column Order"
```

I know this is what we had before but as it is now referred to as "Fixed Columns and Direction" I think we should change this to 
```suggestion
JGLOBAL_MULTI_COLUMN_ORDER_LABEL="Multi Column Direction"
```

---

done